src/utils/create-file.ts

```diff
@@ -2,9 +2,9 @@ import { writeFile } from 'fs'
 import { format } from 'path'
 import { promisify } from 'util'
 
-export async function createFile(name: string, content: string, extension: string): Promise<string> {
+export async function createFile(outputDir: string, name: string, content: string, extension: string): Promise<string> {
   const writeF = promisify(writeFile)
-  const fullPath = format({ name: name, ext: extension })
+  const fullPath = format({ dir: outputDir, name: name, ext: extension })
```

Comment:
Passing an object with the arguments of `format` is better than passing them as string.

As I said before, I don't think that this function is useful. You can replace it with these two lines of code:
```javascript
const path = format( { ... } ); // object receives the current function arguments
await promisify( fs.writeFile )( path, content );
```

Furthermore, it's necessary to pass the adopted `fs` library as an additional argument, in order to be testable with different filesystem.

I suggest to replace the code where it is used with the two lines above.